projects/storefrontapp-e2e/src/page-objects/search-results.po.ts

```diff
@@ -44,7 +44,9 @@ export class SearchResultsPage extends AppPage {
     by.css('.cx-product-search__sorting--top cx-product-view > div > div')
   );
 
-  readonly facet: ElementFinder = this.page.element(by.css('#facetCheck0_0'));
+  readonly facet: ElementFinder = this.page.element(
+    by.css('.cx-search-facet-checkbox')
+  );
 
```

Comment:
```suggestion
  readonly facets: ElementArrayFinder = this.page.all(
    by.css('.cx-search-facet-checkbox')
  );
```

Reply:
Currently we navigating bunch of elements with the same class. So let's put them into `ElementArrayFinder`.

![e2e_selector](https://user-images.githubusercontent.com/6958810/48195018-e1337400-e34e-11e8-8397-03ced3a3a33f.png)

Reply:
Also we can create at the end of the file the method which returns only one element with `n` index from array.

```
 async getSingleFilterFacet(id: number) {
    return this.facets.get(id);
  }
```

Reply:
Then in `product-search.e2e-spec.ts:71` let's use method which click only one facet. There is no necessary to click all of 114 facets.

` await searchResults.getSingleFilterFacet(0).click();`